test/firtool/chirrtl.fir

```diff
@@ -11,29 +11,31 @@ circuit test: %[[{
   }]]
   module test:
     input p: UInt<1>
-    input addr: UInt<4>
+    input addr1: UInt<4>
+    input addr2: UInt<4>
+    input addr3: UInt<4>
     input clock: Clock
     input data: UInt<8>
     output out0: UInt<8>
     output out1: UInt<8>
 
-    ; CHECK: testmem_combMem testmem_ext (
```

Comment:
Dropping the suffix `combMem` is reasonable, but maybe internally there is a hacky logic that detects combinatorial memories by checking the suffix. I think it's totally OK to drop the suffix at will but it would be nice to check beforhand. It seems the suffix is introduced by https://github.com/llvm/circt/commit/2167a06a2318f0839e340cadeb8038f82acdd5ed, @prithayan do you remember if that was requirement?

Reply:
Makes sense to drop the `combMem`, I don't think any tool is relying on that.